docs/output.md

```diff
@@ -35,7 +229,103 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
 
 ![MultiQC - FastQC adapter content plot](images/mqc_fastqc_adapter.png)
 
-> **NB:** The FastQC plots displayed in the MultiQC report shows _untrimmed_ reads. They may contain adapter sequence and potentially regions with low quality.
+:::note
+The FastQC plots displayed in the MultiQC report shows _untrimmed_ reads. They may contain adapter sequence and potentially regions with low quality.
+:::
+
+### Nanocomp
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `batch_qcs/`
+  - `nanocomp/`
+    - `fastq/` and `bam/`
+      - `NanoComp_*.log`: This is the log file detailing the nanocomp run.
+      - `NanoComp-report.html` - This is browser-viewable report that contains all the figures in a single location.
+      - `*.html`: Nanocomp outputs all the figures in the report as individual files that can be inspected separately.
+      - `NanoStats.txt`: This file contains quality control statistics about the dataset.
+
+</details>
+
+![Nanocomp](images/nanocomp_1.png)
+![Nanocomp](images/nanocomp_2.png)
+
+[Nanocomp](https://github.com/wdecoster/nanocomp) compares multiple runs of long read sequencing data and alignments. It creates violin plots or box plots of length, quality and percent identity and creates dynamic, overlaying read length histograms and a cumulative yield plot
+
+### Nanoplot
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `<sample_identifier>/`
+  - `qc/`
+    - `nanoplot/`
+      - `pre_trim/` and `post_trim/` and `post_extract/`
+        - `NanoPlot_*.log`: This is the log file detailing the nanoplot run
+        - `NanoPlot-report.html` - This is browser-viewable report that contains all the figures in a single location.
+        - `*.html`: Nanoplot outputs all the figures in the report as individual files that can be inspected separately.
+        - `NanoStats.txt`: This file contains quality control statistics about the dataset.
```

Comment:
@LilyAnderssonLee : Nanostat/Nanocomp has been a feature that we have been wanting to add to the pipeline's MultiQC report (see issue #15). It was just very recently (as in last week, so very fresh) added to the latest version of MultiQC. We are evaluating it and will add to the first release if possible. If not, it will be added to the first patch. Thanks!

Reply:
We've upgraded MultiQC as well with commit bed7b6e. The main reason its included is it provides a different way to visualize the quality of the data as well as some additional stats not provided by FastQC (such as median quality, read n50, etc.)

Reply:
It sounds good.